src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -56,6 +56,11 @@ public interface IChartEditorPresenter : IPresenter<IChartEditorView>,
       /// </summary>
       void AddDataRepositories(IEnumerable<DataRepository> dataRepositories);
 
+      /// <summary>
+      /// sets the group row format in the gridView of the DataBrowserView to the specified string.
+      /// </summary>
+      void SetGroupRowFormat(string format);
```

Comment:
I do not know what you need this at this level of present.er Don't we have access to the underlying DAtaBrowser presenter with a property get?

Reply:
Not really. We have access to the ChartEditorPresenter (meaning we do not have to go through the ChartPresenter too), but the DataBrowserPresenter is private to the ChartEditorPresenter. If you think it is better, of course I can expose the DataBrowserPresenter....